policies/project-director-election-process.md

```diff
@@ -0,0 +1,60 @@
+* Ensure participants feel positive about their interactions and the execution of the process.
+
+## Description
+
+### Setup
+
+* The council selects a facilitator by consent:
+    * Any team member can suggest a facilitator or volunteer themselves.
+    * The facilitator must not be a nominee in the election and must be someone everyone trusts to set aside personal preferences during the election process.
+    * **Aims of the facilitator role**
+        * Clearly documenting and communicating the process and announcing and supporting its execution.
+        * Tracking, reviewing, and documenting the effectiveness of the process.
+* Review or Define Role Description ([example](https://github.com/rust-lang/rfcs/pull/3392#issuecomment-1505697944))
+    * Aims
+    * Time Investment
+    * Responsibilities
+    * Qualifications
+    * Term
+
+### Pre-Meeting Coordination
+
+* Gather nominations
+    * Council representatives collaborate with their subteams to collect nominations on behalf of their branch of the organization tree.
+    * Subteams can utilize any process they prefer, but we recommend following a similar approach to the process outlined in the "Election Meeting" section below
+* Share nominations publicly
+    * Each team representative must publicly share their team's nominations at least ten days before the final election meeting. This allows other teams to share feedback or objections about candidates their team didn't initially consider with their representative.
+    * This 10-day period is equivalent to the Change Round in the "Election Meeting" step; representatives may change their nominations in advance of the meeting
+
+### Election Meeting
+
+* Share nominations
+    * Write down nominations first[^1].
+    * Use a round format (e.g. [round format](https://www.sociocracyforall.org/on-rounds/))
+    * Nominations must include supporting reasoning 
+* Change Round
+    * During the change round, every team member will say who they nominate now and share reasons for their change if that applies.
+* Facilitator proposes a candidate:
+    * Based on gathered data, the facilitator suggests a candidate they believe will receive the group's consent.
+* Seek objections to the facilitator's proposal
+* Finalize the selection:
+    * Discuss and attempt to resolve any objections, if possible.
+    * Consent has been reached if the proposed choice has no remaining objections! :tada:
+    * If objections persist and the current proposal reaches an impasse, the facilitator should make a new proposal based on the discussion and feedback received.
+    * Continue seeking consensus and addressing objections until a satisfactory candidate is selected.
+        * Note that this process seeks to produce a candidate everyone is satisfied with and has no objections to. Avoid lengthy debates over preferences not related to objections or unmet requirements.
+* Document and announce the selection
```

Comment:
I believe once consensus is reached, we should hold a formal vote. There are compliance reasons to ensure that the final process be a vote.

Reply:
not opposed, tho I don't want to include that until we've confirmed it is necessary for compliance, cc @abibroom

Reply:
Given how the plan is to have Abi run the draft by legal council once we consent to sharing it with the leadership council for review, I am going to leave this comment unresolved with the understanding that this is non-blocking for the initial consent round and that it is only a blocker for the leadership council consent round.

Reply:
Going to resolve this for now.